Assets/_BForBoss/_Character/Scripts/PlayerSlowMotionBehaviour.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using DG.Tweening;
+using Perigon.Utility;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.Rendering;
+
+namespace Perigon.Character
+{
+    public class PlayerSlowMotionBehaviour : MonoBehaviour
```

Comment:
I'm wondering why you put this in Character instead of the Weapon space?

Reply:
Yeah I was a little on the fence with that, I figured it doesn't really fit neatly with either. Ended up going with Character largely because it fit a really similar code pattern with dashing and so plugged into the first person controller quite well imo

Reply:
yeah that's fair, my personal line of thinking is that bullet time is normally associated with Shooting as opposed to movement. I don't think this will be an issue in the future regarding where we put it.

Reply:
Do you think I should move it to the shooting then?

Reply:
yeah I think it fits more in shooting.

Reply:
Follow up question, if I move it to shooting, how should I go about getting input actions and enabling them? Would that plug into the EquipmentManager?

Reply:
yeah, should be the similiar way to how shooting is implemented.

Reply:
Cool cool I'll take a look